docs/Install/Execute_Task.md

```diff
@@ -0,0 +1,25 @@
+# Execute task
+
+User can use the **Execute Task** to initiate execution of various types of Swarm tasks on a task runner and see the progress of the execution of the task.
+
+1.  In the **Task List**, click **Execute**.
+
+2.  Execute the task by providing the following details.
+
+    The following screenshot shows typical training task’s Execute panel.
+
+    ![Execute Task](GUID-72D32A91-39AB-44C8-8854-411A5AC6B0C2-high.png)
+
+    User can select either **All Peers** or **\# of Peers** to execute the task. If user selects **All Peers** option, it schedules the task on all the SWOPs listening on the selected Task Runner.
+
+3.  Enter unique description and select **Save Experiment** to save a particular execution of an experiment.
```

Comment:
Add to point 3:
If this option is not chosen, then the details of this particular experiment is not saved in experiment tracking table.